README.md

```diff
@@ -5,8 +5,27 @@
 
 ## Getting started
 
-TODO
+The goal of the library is to help generate better OpenApi document with less effort.
 
+To achieve this, the library offers an opinionated configuration of OpenAPI document generation and SwaggerUI.
+
+As such, we provide the following features:
+
+- Display OperationId in SwaggerUI
+- (Optional) Fallback to use controller name as OperationId when there is no OperationId explicitly defined for the endpoint.
+
+### How to use it
```

Comment:
Usually in our other libraries, the start of the readme is for context about the lib and then the `Getting Started` section is about how to use the library, you can take a look at this one for reference https://github.com/gsoft-inc/wl-extensions-mongo